### PR TITLE
Add  Slack Service to WDIO 

### PR DESCRIPTION
**Pull request by carmenmitru**
I created a service that sends notifications about tests.
I also created a README in my repo with the installation steps and how to configure it .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ If you're looking for issues to help out with, check out [the issues labelled "g
 - [@wdio/selenium-standalone-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-selenium-standalone-service) - A WebdriverIO service that automatically sets up a selenium standalone server
 - [@wdio/browserstack-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-browserstack-service) - A WebdriverIO service that provides a better integration into Browserstack
 - [@wdio/crossbrowsertesting-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-crossbrowsertesting-service) - A WebdriverIO service that provides a better integration into CrossBrowserTesting
+- [@wdio/slack-service](https://github.com/carmenmitru/wdio-slack-service) - A WebdriverIO service which sends notifications of test results to Slack.
```
Comment (baruchvlz) [inline README.md]: I'm not 100% sure how much this matters to the governance, but I think community projects should be named `wdio-name-type` (ie, `wdio-slack-service).

**Comment by baruchvlz** [comment]
Please add this to the [cli options](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cli/src/constants.js#L66) as well.

File: README.md (hunk 1 above)
Comment (carmenmitru) [reply README.md]: @balukov I've seen that the new packages for v5 are named liek this **@wdio/name-type**.
**wdio-name-type** is used for th packages for V4.

**Comment by BorisOsipov** [comment]
@baruchvlz @carmenmitru I am not sure that third party service should be mentioned in readme. As I see it is only for wdio team maintened packages.

**Comment by BorisOsipov** [comment]
@christian-bromann @mgrybyk please confrim my assumptions

**Comment by carmenmitru** [comment]
@balukov I added Slack service to CLI options

**Comment by carmenmitru** [comment]
@BorisOsipov Should I delete it from README?

File: README.md (hunk 1 above)
Comment (carmenmitru) [reply README.md]: @BorisOsipov Should I delete it from README?

**Comment by christian-bromann** [comment]
The readme should just contain names of packages that are in /packages. Please add your service to the CLI option instead.

**Comment by carmenmitru** [comment]
@christian-bromann  I deleted from README.

**Comment by christian-bromann** [comment]
Can you please sign the CLA?

**Comment by carmenmitru** [comment]
@christian-bromann  How can I  sign the CLA? Sorry it's the first time I a newbie in open source contribution.

**Comment by carmenmitru** [comment]
@christian-bromann  This is the CLA :
<a href="https://www.clahub.com/agreements/carmenmitru/webdriverio">sign the Contributor License Agreement</a>.

**Comment by mgrybyk** [comment]
Make sure your commit email matches github email

**Comment by carmenmitru** [comment]
@mgrybyk yes it is the same

File: README.md [hunk 2]
```diff
@@ -83,6 +83,7 @@ If you're looking for issues to help out with, check out [the issues labelled "g
 - [@wdio/browserstack-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-browserstack-service) - A WebdriverIO service that provides a better integration into Browserstack
 - [@wdio/crossbrowsertesting-service](https://github.com/webdriverio/webdriverio/tree/master/packages/wdio-crossbrowsertesting-service) - A WebdriverIO service that provides a better integration into CrossBrowserTesting
 
+
```
Comment (christian-bromann) [inline README.md]: Let's remove this line

```suggestion
```

File: packages/wdio-cli/src/constants.js [hunk 3]
```diff
@@ -77,6 +77,7 @@ export const SUPPORTED_PACKAGES = {
         { name: 'appium', value: '@wdio/appium-service$--$appium' },
         { name: 'firefox-profile', value: '@wdio/firefox-profile-service$--$firefox-profile' },
         { name: 'crossbrowsertesting', value: '@wdio/crossbrowsertesting-service$--$crossbrowsertesting' },
+        { name: 'slack', value: '@wdio/slack-service$--$slack' },
```
Comment (christian-bromann) [inline packages/wdio-cli/src/constants.js]: This is not an internal package as it comes from you (a 3rd party contributor). Therefor let's move it into the `external` section.

**Comment by christian-bromann** [comment]
Also you can't release your package under the `@wdio` namespace. You will have to call it `wdio-slack-service`. Also how is this service different from https://github.com/kmnaid/wdio-slack-reporter#readme ?

**Comment by carmenmitru** [comment]
@christian-bromann that service is not compatible with WDIO5

**Comment by carmenmitru** [comment]
@christian-bromann  I added it.

File: packages/wdio-cli/src/constants.js [hunk 4]
```diff
@@ -77,11 +77,13 @@ export const SUPPORTED_PACKAGES = {
         { name: 'appium', value: '@wdio/appium-service$--$appium' },
         { name: 'firefox-profile', value: '@wdio/firefox-profile-service$--$firefox-profile' },
         { name: 'crossbrowsertesting', value: '@wdio/crossbrowsertesting-service$--$crossbrowsertesting' },
+
```
Comment (christian-bromann) [inline packages/wdio-cli/src/constants.js]: This line is causing a linter error, please remove

**Comment by carmenmitru** [comment]
@christian-bromann  I removed it. Thanks!

**Comment by balukov** [comment]
> @balukov I added Slack service to CLI options

Thank you, but why me? =)

**Pull request merged**